x-pack/plugin/async-search/src/test/java/org/elasticsearch/xpack/search/RestSubmitAsyncSearchActionTests.java

```diff
@@ -72,8 +72,8 @@ public void testParameters() throws IOException {
         boolean requestCache = randomBoolean();
         doTestParameter("request_cache", Boolean.toString(requestCache), requestCache,
                 r -> r.getSearchRequest().requestCache());
-        Integer batchReduceSize = randomIntBetween(2, 50);
-        doTestParameter("batched_reduce_size", Integer.toString(batchReduceSize), batchReduceSize,
+        int batchedReduceSize = randomIntBetween(2, 50);
+        doTestParameter("batched_reduce_size", Integer.toString(batchedReduceSize), batchedReduceSize,
                 r -> r.getSearchRequest().getBatchedReduceSize());
```

Comment:
it would be cleaner to also clean up the channels being tracked due to the dispatchRequest calls from this test. I gave it a try though and it was not trivial, and the same should be done in RestCancellableNodeClientTests where we actually assume that we may have channels in the map already, hence we only check the delta since the test has started. I wonder if it's worth looking into cleaning the channels up after both tests have executed, or not.